tfrecorder/check_test.py

```diff
@@ -120,6 +120,7 @@ def test_valid_records(self, mock_fn):
 
       # Check output CSV
       actual_df = pd.read_csv(actual_csv)
+      print(actual_df)
```

Comment:
omit

Reply:
I really need to add a check for this in my presumit.  I do it all the time. :(

Ack, removed.